docs/workflows/transitions/postprocess/postprocess/utils.py

```diff
@@ -35,3 +35,16 @@ def convert_predictions_to_v2(predictions):
             result[label] = sorted(result[label], key=lambda p: -p['confidence'])
 
     return result
+
+
+def to_validated_format(predictions):
+    def add_metainfo(el):
+        return {'isEdited': False, 'automated': True, **el}
+
+    def top1_pred(val):
+        if 'value' in val[0].keys():
```

Comment:
Will this test fail if we have a line items field that is named 'value'? It think we should have a more robust and reusable way of determining whether a prediction is a line prediction a normal field or a multivalue field.